src/qt/rpcconsole.cpp

```diff
@@ -958,6 +959,18 @@ void RPCConsole::updateNetworkState()
     }
 
     ui->numberOfConnections->setText(connections);
+
+    // localAddressess
+    std::string localAddresses="";
+    LOCK(g_maplocalhost_mutex);
+    for (const std::pair<const CNetAddr, LocalServiceInfo> &item : mapLocalHost)
+        localAddresses += item.first.ToString() + ":" + std::to_string(item.second.nPort) + ", ";
```

Comment:
to fix the linter issue arrising from `std::to_string(item.second.nPort), and keeping in mind that `locale_addresses` is now a QString: 

```suggestion
        local_addresses += QString::fromStdString(item.first.ToString()) + ":" + QString::number(item.second.nPort) + ", ";
```

`QString::number` fixes the linter issue, from [Qt Docs](https://doc.qt.io/qt-5/qstring.html#number): "The formatting always uses [QLocale::C](https://doc.qt.io/qt-5/qlocale.html#Language-enum), i.e., English/UnitedStates"

Reply:
Thank you very much. Greatly appreciate. Will do the changes & will commit.